1684231244749-create-tables.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
what's the purpose of this file?
Is it necessary to commit this to the codebase? or is it a temporary file?